app/bundles/EmailBundle/Model/EmailModel.php

```diff
@@ -1057,6 +1057,24 @@ public function getQueuedCounts(Email $email, $includeVariants = true)
         return $queued;
     }
 
+    public function getDeliveredCount(Email $email, bool $includeVariants = false): int
+    {
+        $emailIds = ($includeVariants && ($email->isVariant() || $email->isTranslation())) ? $email->getRelatedEntityIds() : [$email->getId()];
+
+        $statRepo = $this->getStatRepository();
+
+        /** @var \Mautic\LeadBundle\Entity\DoNotContactRepository $dncRepo */
+        $dncRepo = $this->em->getRepository(DoNotContact::class);
+
+        $failedCount    = (int) $statRepo->getFailedCount($emailIds);
+        $bouncedCount   = (int) $dncRepo->getCount('email', $emailIds, DoNotContact::BOUNCED);
+        $sentCount      = (int) $email->getSentCount($includeVariants);
```

Comment:
I can add similar comment to these counts as I just wrote in https://github.com/mautic/mautic/pull/11945#pullrequestreview-1360995754

Our clients have hundreds of millions of rows in the email_stats table. This will make Mautic unusable for them.